examples/hosts/app-integration/external-data/src/view/debugView.tsx

```diff
@@ -158,14 +156,151 @@ function debugResetExternalData(): void {
+    const { task } = props;
+
+    const IdChangeHandler = (e: React.SyntheticEvent<HTMLInputElement>): void => {
+        task.id = e.currentTarget.value;
+    };
+    const NameChangeHandler = (e: React.SyntheticEvent<HTMLInputElement>): void => {
+        task.name = e.currentTarget.value;
+    };
+    const PriorityChangeHandler = (e: React.SyntheticEvent<HTMLInputElement>): void => {
+        task.priority =  Number.parseInt(e.currentTarget.value, 10);
+    };
+
+    return (
+        <tr>
+            <td><input defaultValue={ task.id }  style={{ width: "30px" }} onChange={ IdChangeHandler }></input></td>
+            <td><input defaultValue={ task.name } style={{ width: "200px" }} onChange={ NameChangeHandler }></input></td>
+            <td><input defaultValue={ task.priority } type="number" style={{ width: "50px" }} onChange={ PriorityChangeHandler }></input></td>
+        </tr>
+    );
+};
+
+class ExternalDataTask {
+    public id: string;
+    public name: string;
+    public priority: number;
+    public constructor(id: string, name: string, priority: number) {
+        this.id = id;
+        this.name = name;
+        this.priority = priority;
+    }
+}
+/**
+ * A tabular, editable view of the task list.  Includes a save button to sync the changes back to the data source.
+ */
+export const TaskListView: React.FC<IDebugViewProps> = () => {
+    const [externalData, setExternalData] = useState({});
+    useEffect(() => {
+        // HACK: Once we have external changes triggering the appropriate Fluid signal, we can simply listen
+        // for changes coming into the model that way.
+        // For now, poll the external service directly for any updates and apply as needed.
+        async function pollForServiceUpdates(): Promise<void> {
+            try {
+                const response = await fetch(
+                    `http://localhost:${customerServicePort}/fetch-tasks`,
+                    {
+                        method: "GET",
+                        headers: {
+                            "Access-Control-Allow-Origin": "*",
+                            "Content-Type": "application/json",
+                        },
+                    }
+                );
+
+                const responseBody = await response.json() as Record<string, unknown>;
+                const newData = responseBody.taskList as TaskData;
+                if(newData !== undefined && !isEqual(newData,externalData)) {
+                    console.log("APP: External data has changed. Updating local state with:\n", newData)
+                    setExternalData(newData);
+                }
+            } catch(error) {
+                console.error(`APP: An error was encountered while polling external data:\n${error}`);
+            }
+        }
+
+        // Run once immediately to run without waiting.
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
+        pollForServiceUpdates();
```

Comment:
```suggestion
        pollForServiceUpdates().catch(console.error);
```
Nit. So failures during the request at least get printed somewhere, and I think this should make the eslint-disable unnecessary.